README.md

```diff
@@ -174,7 +174,7 @@ Where PROJECT_ID is your project ID
 This will instruct the debugger CLI to create and use a database with the name
 `PROJECT_ID-cdbg`
 
-1. Run `python3 cli/cli.py init` in the cloned `snapshot-debugger`
+1. Run `python3 -m cli init` in the cloned `snapshot-debugger`
```

Comment:
Per offline discussion, cli is too much of a generic module name in this context.  It should either be namespaced or be given a more specific name for disambiguation.

Reply:
I've switched to using `snapshot_dbg_cli` as the package name.

PTAL

Reply:
LGTM